src/TestSuite/IntegrationTestCase.php

```diff
@@ -211,10 +211,17 @@ public function session(array $data)
      *
      * @param string $name The cookie name to use.
      * @param mixed $value The value of the cookie.
+     * @param string|bool|null $encryption By default, the CookieComponent
+     *   isn't be used. If specified the encryption mode like 'aes' or false,
+     *   the CookieComponent is used to encrypt and json_encode if array.
```

Comment:
What if encrypted cookies were set through a different method? I think that might result in an easier to read API. Boolean flags are harder to grok than distinct method names.